synapse/rest/client/login.py

```diff
@@ -420,25 +422,51 @@ async def _do_jwt_login(
                 403, "Token field for JWT is missing", errcode=Codes.FORBIDDEN
             )
 
-        import jwt
+        jwt = JsonWebToken([self.jwt_algorithm])
```

Comment:
I think you'll need to add the imports from authlib (lines +29, +30) here.

Reply:
Thanks for this hint. I'll move the imports and run tests for "all extras" and "no extras"